typescript/introduction/tsc-variable-declaration/tsc-let.md

```diff
@@ -0,0 +1,91 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Introducing `let`
```

Comment:
this insight doesn't talk too much about using `let` in the context of typescript - to me it loos like an ES6 insight

Reply:
True but if we only include insights related to specifics of TS without explaining any of the syntax it shares with JS it would be unfriendly to beginners right?

Reply:
depends on how you look at the purpose of the content 

- do we want to present typescript features 

or

- do we want to explain how to write programs in typescript 

if it's the latter we then need to introduce way more concepts, but imo this is not scalable at all. this is something that needs fixing from our business logic, not from the content

Reply:
the same argument can be applied to react - if people want to learn to react from scratch in our app we should present all these canonic JS features

Reply:
Yeah, it's kind of a tough spot. 

Note that React/JS isn't exactly the same situation as TS/JS as React is just some JS code in a particular structure while TS is a whole new language that happens to use, as its base, the same rules as JavaScript